qa/paqa_macros.h

```diff
@@ -39,17 +42,26 @@ extern int paQaNumFailed;
             printf("ERROR at %s:%d, (%s) %s (%s), %d %s %d\n", \
                 __FILE__, __LINE__, #_a, #_opn, #_b, mA, #_opn, mB ); \
             paQaNumFailed++; \
-            goto error; \
+            _on_error; \
         } \
     } while(0)
 
+#define ASSERT_AB(_a, _b, _op, _opn) CHECK_AB(_a, _b, _op, _opn, goto error)
 #define ASSERT_EQ(_a, _b) ASSERT_AB(_a, _b, ==, !=)
 #define ASSERT_NE(_a, _b) ASSERT_AB(_a, _b, !=, ==)
 #define ASSERT_GT(_a, _b) ASSERT_AB(_a, _b, >, <=)
 #define ASSERT_GE(_a, _b) ASSERT_AB(_a, _b, >=, <)
 #define ASSERT_LT(_a, _b) ASSERT_AB(_a, _b, <, >=)
 #define ASSERT_LE(_a, _b) ASSERT_AB(_a, _b, <=, >)
 
+#define EXPECT_AB(_a, _b, _op, _opn) CHECK_AB(_a, _b, _op, _opn, )
```

Comment:
I'm not sure which version of the C standard supports zero-token macro arguments.